.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "thundra-demo-todo-app-java"]
+	path = thundra-demo-todo-app-java
+	url = git@github.com:thundra-io/thundra-demo-todo-app-java.git
+[submodule "thundra-demo-localstack-nodejs"]
+	path = thundra-demo-localstack-nodejs
+	url = git@github.com:thundra-io/thundra-demo-localstack-nodejs.git
+[submodule "thundra-demo-localstack-java"]
+	path = thundra-demo-localstack-java
+	url = git@github.com:thundra-io/thundra-demo-localstack-java.git
+[submodule "thundra-debugger-demo"]
```

Comment:
I think, we can also add `thundra-demo-localstack-java` in advence as it will be merged very soon

Reply:
Added in advance. I'll fetch the upstream and update the linked commit once it's merged.